src/config.c

```diff
@@ -211,6 +211,16 @@ int load_config(struct vpn_config *cfg, const char *filename)
 			cfg->user_cert = strdup(val);
 		} else if (strcmp(key, "user-key") == 0) {
 			cfg->user_key = strdup(val);
+		} else if (strcmp(key, "insecure-ssl") == 0) {
+			int insecure_ssl = strtob(val);
+			if (insecure_ssl < 0) {
+				log_warn("Bad insecure-ssl in config file: \"%s\".\n",
+				         val);
+				continue;
+			}
+			cfg->insecure_ssl = insecure_ssl;
+		} else if (strcmp(key, "cipher_list") == 0) {
```

Comment:
Actually that's what I wanted too, mixed it up with the variable and shamefully admitting I only tried if command-line option worked!